### DISCUSSION TITLE
V0 - Failed to fetch components from registry.

### DISCUSSION THREAD

**Discussion by tiagoefreitas** [vercel/community#5234] [Bug Report]
### Page to Investigate

https://v0.dev/docs#integrating-generated-code-into-your-nextjs-app

### Steps to Reproduce

If I do npx v0 add on a private v0 id I get 
Error: Component XXX not found.
 at file:///Users/XXX/.npm/_npx/7a9c409306f58534/node_modules/v0/dist/index.js:2:4548
Failed to fetch components from registry.

**Comment by amyegan** [comment]
Hey Tiago! Did you include TypeScript, Tailwind, and ESLint when creating your Next.js app? I wasn't able to reproduce this error with a fresh project following the steps from that guide.

**Reply by tiagoefreitas** [reply]
It only happens if the v0 design is private, it works if I make it public. But it would be nice if the CLI would authorize me in the browser and work for private designs

**Comment by RicardoRickTron** [comment]
same problem to me. my v0 design is private and I need to keep private, any solution? maybe I need to authenticate before do a fetch?

**Comment by Yassinebouhlel** [comment]
anyone able to solve this ? I'am stuck in the same error

**Comment by ngoiyaeric** [comment]
unable to deploy v0 locally getting generic next js page with next links

**Reply by amyegan** [reply]
Hey @ngoiyaeric. I wasn't able to replicate this behavior just now. If it's still a problem for you, please [start a new discussion](https://github.com/vercel/community/discussions/new) and share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so I can take a closer look 🙏
